ckanext/cfpb_extrafields/fanstatic/popover.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
```

Comment:
mostly copied from http://docs.ckan.org/en/latest/theming/javascript.html#ajax-event-handling-and-ckan-s-javascript-sandbox